From 3c31924598630db24c5a972fcf53c8ddc0f3ad4a Mon Sep 17 00:00:00 2001
From: Wren Turkal <wt@penguintechs.org>
Date: Sun, 29 Jun 2025 02:05:20 -0700
Subject: [PATCH] Direct logs to systemd journal

The logs for the systemd example should be directed to the systemd
journal.
---
 contrib/systemd/spacenavd.service | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/contrib/systemd/spacenavd.service b/contrib/systemd/spacenavd.service
index f0e3961..2632fb8 100644
--- a/contrib/systemd/spacenavd.service
+++ b/contrib/systemd/spacenavd.service
@@ -1,12 +1,10 @@
 [Unit]
 Description=3Dconnexion Input Devices Userspace Driver
-After=syslog.target
 
 [Service]
-Type=forking
+Type=exec
 PIDFile=/var/run/spnavd.pid
-ExecStart=/usr/local/bin/spacenavd
-StandardError=syslog
+ExecStart=/usr/local/bin/spacenavd -d -l /dev/stdout
 
 [Install]
 WantedBy=graphical.target
